staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo_test.go

```diff
@@ -45,7 +45,9 @@ func TestGetAPIRequestInfo(t *testing.T) {
 
 		// resource paths
 		{"GET", "/api/v1/namespaces", "list", "api", "", "v1", "", "namespaces", "", "", []string{"namespaces"}},
-		{"GET", "/api/v1/namespaces/other", "get", "api", "", "v1", "other", "namespaces", "", "other", []string{"namespaces", "other"}},
+		{"GET", "/api/v1/namespaces/other", "get", "api", "", "v1", "", "namespaces", "", "other", []string{"namespaces", "other"}},
+		{"GET", "/api/v1/watch/namespaces/other", "watch", "api", "", "v1", "", "namespaces", "", "other", []string{"namespaces", "other"}},
+		{"GET", "/api/v1/watch/namespaces", "watch", "api", "", "v1", "", "namespaces", "", "", []string{"namespaces"}},
```

Comment:
This test case is fine (but already works at HEAD without any change to staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo.go).